README.md

```diff
@@ -157,6 +157,15 @@ jobs:
           # For python source code only. This is checked on a per-file basis
           only_code: true
 ```
+## Using duplicate-code-check with pre-commit
+To use Duplicate Code Detection Tool as a pre-commit hook with [pre-commit](https://pre-commit.com/) add the following to your `.pre-commit-config.yaml` file:
+```yaml
+-   repo: https://github.com/platisd/duplicate-code-detection-tool.git
+    rev: ''  # Use the sha / tag you want to point at
+    hooks:
+    -   id: duplicate-code-detection
+```
+> **_NOTE:_** that this repository sets args: `-f`, if you are configuring duplicate-code-detection-tool using args you'll want to include either `-f` (`--files`) or `-d` (`--directories`).
```

Comment:
Can you please elaborate/explain what you mean by the note?
Perhaps I would understand if I were more familiar with pre-commit hooks or it's just that my brain is stuck right now. :shrug: 

Is it that the tool will be run only for the files changed in the latest commit or?

Reply:
> Is it that the tool will be run only for the files changed in the latest commit or?

Yes, pre-commit automatically appends the filenames of the staged files so if we invoke your tool with `-f` which is the default it will work out of the box.
If you override the argument(s) by adding your own, the default argument will not be supplied anymore.